packages/tools/src/utilities/math/aabb/distanceToPointSquared.ts

```diff
@@ -0,0 +1,53 @@
+import { Types } from '@cornerstonejs/core';
+
+/**
+ * Calculates the distance of a point to an AABB using 2D Box SDF (Signed Distance Field)
+ *
+ * The SDF of a Box
+ * https://www.youtube.com/watch?v=62-pRVZuS5c
+ *
+ * @param aabb - Axis-aligned bound box
+ * @param point - 2D point
+ * @returns The closest distance between the 2D point and the AABB
+ */
+export default function distanceToPointSquared(
+  aabb: Types.AABB2,
+  point: Types.Point2
+): number {
+  const aabbWidth = aabb.maxX - aabb.minX;
+  const aabbHeight = aabb.maxY - aabb.minY;
+  const aabbSize = [aabbWidth, aabbHeight];
+  const aabbCenter: Types.Point2 = [
+    aabb.minX + aabbWidth / 2,
+    aabb.minY + aabbHeight / 2,
+  ];
+
+  // Translates the AABB and the point using AABB center as the new origin
+  aabb = {
+    minX: aabb.minX - aabbCenter[0],
+    minY: aabb.minY - aabbCenter[1],
+    maxX: aabb.maxX - aabbCenter[0],
+    maxY: aabb.maxY - aabbCenter[1],
+  };
+
+  // The point is not only translated but also mirroed to the first quadrant to simplify the math
+  point = [
+    Math.abs(point[0] - aabbCenter[0]),
+    Math.abs(point[1] - aabbCenter[1]),
+  ];
+
```

Comment:
you are modifying the function argument, which might have side effect

Reply:
- Renamed the point to `translatedPoint`.
- Removed the lines that updated AABB's origin because we don't need it anymore because `aabbSize * 0.5` gives what is needed.